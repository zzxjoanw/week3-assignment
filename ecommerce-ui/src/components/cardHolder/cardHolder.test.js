import CardHolder from "./cardHolder";
import renderer from 'react-test-renderer';
import { shallow, mount } from 'enzyme'
import React from 'react';

describe('CardHolder test', () => {
    it('renders without crashing', () => {
        expect(<CardHolder />).toMatchSnapshot();
    });
    
    const sites = ['Denver', 'Budapest', 'Tokyo'];
    it('matches even if received contains additional elements', () => {
      expect(['Denver', 'Budapest', 'Tokyo']).toEqual(expect.arrayContaining(sites));
    });

    it('does not match if received does not contain expected elements', () => {
      expect(['Denver', 'Budapest']).not.toEqual(expect.arrayContaining(sites));
    });
  });