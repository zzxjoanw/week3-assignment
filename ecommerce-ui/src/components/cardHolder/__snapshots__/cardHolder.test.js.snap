// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CardHolder test renders without crashing 1`] = `<CardHolder />`;
